pkg/auth/webapp/preview.go

```diff
@@ -1,15 +1,12 @@
 package webapp
 
-import "net/http"
-
-const PreviewQueryKey = "x_preview"
-
-type PreviewMode string
-
-const (
-	PreviewModeInline = "inline"
+import (
+	"net/http"
+	"strings"
 )
 
-func IsPreviewModeInline(r *http.Request) bool {
-	return r.Method == http.MethodGet && r.URL.Query().Get(PreviewQueryKey) == PreviewModeInline
+const InlinePreviewPathPrefix = "/preview"
```

Comment:
It should be `/preview/`. So it will not match `/preview_something_added_in_the_future`.

Reply:
Updated here https://github.com/authgear/authgear-server/pull/4396/commits/6e163cc7098ae8607939a0af458244ce47ccc84e#diff-bf07c7b0ae1992e2a9f6495b630e067dc01e6900cfdbb787d88305e174f042f0R8